packages/nx-plugin/package.json

```diff
@@ -73,7 +72,6 @@
     "vitest": "0.30.0"
   },
   "peerDependencies": {
-    "nx": "^15.7.0",
```

Comment:
Why do we remove the peerDep ? aren't we supposed to have nx to use the generator ?